test/e2e/generate_kube_test.go

```diff
@@ -555,6 +556,15 @@ var _ = Describe("Podman generate kube", func() {
 		kube.WaitWithDefaultTimeout()
 		Expect(kube).Should(Exit(0))
 
+		b, err := os.ReadFile(outputFile)
+		Expect(err).To(BeNil())
+		pod := new(v1.Pod)
+		err = yaml.Unmarshal(b, pod)
+		Expect(err).To(BeNil())
+		val, found := pod.Annotations[vol1]
+		Expect(found).To(BeTrue())
+		Expect(strings.HasSuffix(val, "z")).To(BeTrue())
```

Comment:
```suggestion
		Expect(val).To(HaveSuffix("z"))
```
Produces a more meaningful error message.